server/internal/adapter/gql/gqlmodel/convert_fields.go

```diff
@@ -0,0 +1,25 @@
+package gqlmodel
+
+import "github.com/reearth/reearth-cms/server/internal/usecase/interfaces"
+
+func AssetsToGuessSchemaFieldData(data *interfaces.GuessSchemaFieldsData) *GuessSchemaFieldResult {
```

Comment:
```suggestion
func ToGuessSchemaFieldData(data *interfaces.GuessSchemaFieldsData) *GuessSchemaFieldResult {
```
Use the same naming convention